src/actions/classifier.js

```diff
@@ -72,6 +72,13 @@ export function saveAnnotation(task, value) {
   }
 }
 
+export function removeAnnotationValue(task, value) {
+  return (dispatch, getState) => {
+    const workflowID = getState().classifier.currentWorkflowID
+    dispatch(removeState(`classifier.annotations.${workflowID}.${task}`, value))
```

Comment:
Not quite. Let's say the user selected "Nothing here" and then answers "yes" to the question. Will the final annotation have both "yes" and "Nothing here" saved as an answer.